CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
 * Strings dot-syntax template: use `enum`s for namespacing instead of `struct`s.  
   [David Jennes](https://github.com/djbe)
   [#218](https://github.com/AliSoftware/SwiftGen/pull/218)
+* Updated Stencil to [0.7](https://github.com/kylef/Stencil/releases/tag/0.7.0).  
```

Comment:
Yes, although 0.7 includes 0.7.0, 0.7.1, 0.7.2 😉.

I think it's better to link to 0.7 changelog because 0.7.1 and 0.7.2 do not include changes that will affect any user of SwiftGen. These two releases only include bug fixes for problems in 0.7.